lib/bloc/tmdb_event.dart

```diff
@@ -0,0 +1,6 @@
+part of 'tmdb_bloc.dart';
+
+@freezed
+class TmdbEvent with _$TmdbEvent {
+  const factory TmdbEvent.onAppStarted() = _TmdbCloneOnAppStartedEvent;
```

Comment:
please remove the 'clone' from _TmdbCloneOnAppStartedEvent

Reply:
Done